### PR TITLE
Cleanup listeners code

### PR DESCRIPTION
**Pull request by iMoses**

### CODE REVIEW

File: packages/ag-charts-community/src/util/listeners.ts [hunk 1]
```diff
@@ -1,82 +1,51 @@
...
+                listener.handler(...params);
             } catch (e) {
                 Logger.errorOnce(e);
-                results.push(undefined as any);
             }
         }
-        return results;
-    }
-
-    public cancellableDispatch(
```
Comment (alantreadway) [inline packages/ag-charts-community/src/util/listeners.ts:37]: We should probably keep this behaviour here, I can see it being needed in other cases in the future potentially.

**Review by alantreadway — COMMENTED** [review]
Nice cleanup of the listener removal case - I don't think we should push the cancellable dispatch behaviour down to `InteractionManager` though since there maybe other cases in the future where we want to have similar dispatch behaviour.

File: packages/ag-charts-community/src/util/listeners.ts (hunk 1 above)
Comment (iMoses) [reply packages/ag-charts-community/src/util/listeners.ts:37]: let's talk about it offline

Comment (alantreadway) [reply packages/ag-charts-community/src/util/listeners.ts:37]: Discussed offline - the prior implementation was suspect on 2nd look, we'd want to implement it differently anyway.

**Review by alantreadway — APPROVED** [review]
LGTM - thanks, nice cleanup!

**Pull request merged**